### PR TITLE
[docs] Added perplexity for AWQ.

### PR DESCRIPTION
**Pull request by andreyanufr**
### Changes

Added perplexity for AWQ algorithm to documentation.

### Related tickets

125383

### CODE REVIEW

**Review by MaximProshin — APPROVED** [review]

File: docs/compression_algorithms/CompressWeights.md [hunk 1]
```diff
@@ -289,9 +289,9 @@ Here is the perplexity and model size before and after weight compression for di
 
 Here is the word perplexity with data-free and data-aware mixed-precision INT4-INT8 weight compression for different language models on the [wikitext dataset](https://arxiv.org/pdf/1609.07843.pdf).
 `data` suffix refers to the data-aware mixed-precision.
-
+`data_awq` suffix refers to the data-aware mixed-precision with modified [AWQ](https://arxiv.org/abs/2306.00978) algorithm.
```
Comment (ljaljushkin) [inline docs/compression_algorithms/CompressWeights.md:292]: Maybe it makes sense to add somewhere about specific pattern of layers in order to apply AWQ.

Comment (andreyanufr) [reply docs/compression_algorithms/CompressWeights.md:292]: Done

File: docs/compression_algorithms/CompressWeights.md [hunk 2]
```diff
@@ -302,42 +303,72 @@ Here is the word perplexity with data-free and data-aware mixed-precision INT4-I
         <td>11.87</td>
     </tr>
     <tr>
-        <td>meta-llama/llama-7b-chat-hf</td>
+        <td></td>
```
Comment (AlexKoff88) [inline docs/compression_algorithms/CompressWeights.md:306]: We need baseline metrics for Wikitext as well. @andreyanufr, can you please add them?

Comment (AlexKoff88) [reply docs/compression_algorithms/CompressWeights.md:306]: Can we also add a model footprint for consistency with the previous table? This is also the only differentiation factor that we can publically claim. @andreyanufr, please add it as well.

Comment (AlexKoff88) [reply docs/compression_algorithms/CompressWeights.md:306]: Please also state where `all_layers` option was used if so.

**Review by AlexKoff88 — APPROVED** [review]

**Review by ljaljushkin — APPROVED** [review]

**Pull request merged**